x-pack/test_serverless/functional/test_suites/observability/dataset_quality/home.ts

```diff
@@ -16,13 +17,27 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   ]);
 
   const testSubjects = getService('testSubjects');
+  const synthtrace = getService('svlLogsSynthtraceClient');
+  const to = '2024-01-01T12:00:00.000Z';
 
   describe('Dataset quality home', () => {
     before(async () => {
       await PageObjects.svlCommonPage.loginWithRole('admin');
     });
 
+    after(async () => {
+      await synthtrace.clean();
```

Comment:
nit: We can remove the after block here and this clean can actually be added inside the last it block as there is no common data loaded,only the last `it` block loads the data, which means we would unnecessarily be making a elasticsearch call for the 1st `it`.